test-plan.md

```diff
@@ -93,6 +93,38 @@ The easist way to verify that a project was successfully loaded is to open a .cs
 * In a project that uses globbing (.NET Core), use the VS Code file explorer to add a new file next to the csproj. Intellisense/sighelp/etc should be available in the new file
 * Add a new file and reference a type in it from a different file. Deleting from disk the file containing the referenced type  should produce error messages
 
+#### Razor
```

Comment:
Added this as a sub-item of `Validating C# Extension for VS Code`, is that the right place?

Reply:
Works for me.